docs/UserGuide.md

```diff
@@ -133,14 +136,72 @@ Deletes the specified project from the project list.
 
 Format: `delete project <PROJECT_NAME>`
 
-* The specified project name must exist in the project list.
+- The specified project name must exist in the project list.
+
+Examples:
+
+- `delete project CS2101 Presentation`
+
+Expected output:
+
+- Successful deletion: `<PROJECT_NAME> has been deleted from the project list.`
+- Failed deletion: `Project <PROJECT_NAME> not found: Please make sure the project exists.`
+
+### Assign member to task : `add person`
+
+Assigns a team member to a task within a project.
+
+Format: `add person <PERSON_NAME> /to <TASK_NAME> /in <PROJECT_NAME>`
+
+- The specified project name must exist in the project list.
+- The specified task name must exist in the project's task.
+- The specified member name must be a member of the project team.
+
+Examples:
+
+- `add person Joe /to unit test /in CS2103_TP`
+
+Expected output:
+
+- Successful assignment: `<PERSON_NAME> has been assigned to <PROJECT_NAME>: <TASK_NAME>`
+- Failed assignment (example): `Person <PERSON_NAME> is not a team member.`
+
+### Assign deadline to project : `add deadline`
+
+Assigns a deadline to a project.
+
+Format: `add deadline <DEADLINE> /to <PROJECT_NAME>`
+
+- The specified project name must exist in the project list.
+- The deadline must be in `MMM D YYYY` format
```

Comment:
Should it be 'MM-DD-YYYY'?

Reply:
I think including '-' would make the date format abmiguous.
MMM D YYYY is also a widely known format and should be fine.